{"name": "test_updateaccount", "status": "failed", "statusDetails": {"message": "AssertionError: assert '无效的session' == '成功'\n  - 成功\n  + 无效的session", "trace": "self = <testcase.ceshicase.TestMgmt object at 0x00000204F73317B8>\n\n    def test_updateaccount(self):#编辑账号\n        self.logger = log.Logger\n        base_url = cf.get(\"db\", \"base_url1\") + \"updateAccount\"\n        data ={\"name\": 212300,\n               \"phone\": self.g[\"m\"],\n               \"sex\": 0,\n               \"id\": self.g[\"b\"][\"data\"],\n               \"merchantId\": 158}\n        file = {\"file\": (\"u.jpg\", open('D:/a/u.jpg', \"rb\"), \"image/jpeg\")}\n        r = requests.post(url=base_url, headers=headers, files=file, data=data).json()\n        self.logger.info(r)\n>       assert r[\"resultMessage\"] == \"成功\"\nE       AssertionError: assert '无效的session' == '成功'\nE         - 成功\nE         + 无效的session\n\nceshicase.py:181: AssertionError"}, "attachments": [{"name": "log", "source": "2c08d30f-bbdb-492c-9590-e6072d89b0e8-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "7b1e6d85-e5de-4b8f-a43f-0c77447a44d4-attachment.txt", "type": "text/plain"}], "start": 1626248777904, "stop": 1626248778035, "uuid": "c1637c7f-708e-4195-bf77-02c57b17f132", "historyId": "f6c8aaae73167ecaf309cebaf3ebaecc", "testCaseId": "be386f6c733d1e293c2fac581cf2e035", "fullName": "testcase.ceshicase.TestMgmt#test_updateaccount", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "ceshicase"}, {"name": "subSuite", "value": "TestMgmt"}, {"name": "host", "value": "LAPTOP-1V0QHO6O"}, {"name": "thread", "value": "22600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.ceshicase"}]}